shielder/contract/errors.rs

```diff
@@ -0,0 +1,22 @@
+#![cfg_attr(not(feature = "std"), no_std, no_main)]
```

Comment:
This is redundant as `lib.rs` is already affecting the whole crate`

Reply:
Same comment apply to other files as well

Reply:
removed it